apps/contract/scripts/deploy-auth.sh

```
@@ -0,0 +1,135 @@
+echo "Building Auth Controller Contract..."
+cargo build --target wasm32-unknown-unknown --release --manifest-path ./contracts/auth-contracts/auth-controller/Cargo.toml
+
+echo "Building Account Factory Contract..."
+cargo build --target wasm32-unknown-unknown --release --manifest-path ./contracts/auth-contracts/account-factory/Cargo.toml
+
+echo "✅ All contracts built successfully!"
+
+# Deploy Auth Controller Contract first (since Account needs it)
+echo "📦 Deploying Auth Controller Contract..."
+AUTH_CONTROLLER_WASM_HASH=$(stellar contract upload \
+    --network $NETWORK \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)